tests/SpecTests/CommandExpectations.php

```diff
@@ -102,6 +102,21 @@ public static function fromCrud(array $expectedEvents)
         return $o;
     }
 
+    public static function fromReadWriteConcern(array $expectedEvents)
+    {
+        $o = new self($expectedEvents);
+
+        $o->ignoreCommandFailed = true;
+        $o->ignoreCommandSucceeded = true;
+
+        /* Retryable read spec tests don't include extra commands, e.g. the
```

Comment:
Is this copypasta?

Reply:
Yep. I really need to refactor these tests to avoid copy pasting all this code every time we add a new spec test runner.